istioctl/cmd/describe.go

```diff
@@ -427,6 +428,14 @@ func printPod(writer io.Writer, pod *v1.Pod) {
 			}
 			ports = append(ports, fmt.Sprintf("%d%s (%s)", port.ContainerPort, protocol, container.Name))
 		}
+		// Ref: https://istio.io/latest/docs/ops/deployment/requirements/#pod-requirements
+		if container.Name != "istio-proxy" {
+			if container.SecurityContext != nil && container.SecurityContext.RunAsUser != nil {
+				if *container.SecurityContext.RunAsUser == UID {
+					fmt.Fprintf(writer, "WARNING: Ensure your pods do not run applications as a user with the user ID (UID) value of 1337\n")
```

Comment:
may want to explain why?

Reply:
https://github.com/istio/istio.io/pull/2940

Reply:
I mean for the users. But I guess if istio.io doesn't explain why maybe we don't need to here either

Reply:
I couldn't find any more details why we should not run the application pod with 1337. I think we should add more details instead of not adding anything here.

Reply:
Updated istio.io to include more info https://github.com/istio/istio.io/pull/9462